docs/extra.md

```diff
@@ -0,0 +1,93 @@
+To configure Pelorus to monitor your application, you need 3 information:
+
+* Which Hosting Service(s) for Version Control System (**HSVCS**) your application uses? (Ex.: GitHub)
+* Which Issue Tracker System(s) (**ITS**) your application uses? (Ex.: Jira)
+* Which OpenShift Namespace(s) (**NS**) your application lives in?
+
+To check the list of supported HSVCS and ITS by Pelorus, check [Supported data sources](#supported-data-sources).
+
+## Supported data sources
```

Comment:
Should we tell what user has to do if his data source is not supported?

Reply:
not here, we could tell a bit about out community, what slack channel are we in etc... but it's separate PR and under Contribute section of the doc.